### PR TITLE
Update on-demand-history-recovery-configuration.md

### PR DESCRIPTION
**Pull request by akiaei**
timestamp update for WI https://dev.azure.com/osieng/engineering/_sprints/taskboard/Aurora/engineering/2022/Q2/2022.06.08?workitem=308001
Timestamps that don't specify UTC or local time get interpreted as local time of the adapter.
Ex:  "2022-05-23T11:00" will get interpreted as the same local time as the adapter ("2022-05-23T11:00-07:00") if the adapter is in PDT time (UTC -7:00)

### CODE REVIEW

**Comment by akiaei** [comment]
I'm questioning if this is even needed anymore @zryska what do you think?
I feel like the documentation is clear enough that timestamps get interpreted into UTC

**Comment by zryska** [comment]
> I'm questioning if this is even needed anymore @zryska what do you think? I feel like the documentation is clear enough that timestamps get interpreted into UTC

I agree and I would suggest rephrasing it to make sure we describe options and provide examples - to improve usability of the feature.

File: content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ The PI adapter supports performing history recovery on-demand by specifying star
 Parameter | Type| Description
 ---------|----------|---------
  **Id** | `string` | The Id of the history recovery<br><br> **Note:** You cannot run multiple history recoveries with the same Id.
- **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z.   
- **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z.
+ **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. If not in a UTC format, the timestamp is interpreted as the local time of the adapter.
```
Comment (rmuolic) [inline content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md:30]:
```suggestion
 **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime as UTC in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. <br><br>**Note:** If a timestamp is not specified in UTC, the adapter assumes it is in local time and will interpret the timestamp to UTC. For example, a timestamp of "2022-05-23T11:00" will be interpreted as "2022-05-23T11:00-07:00" if the adapter is in PDT time (UTC -7:00).
```

File: content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ The PI adapter supports performing history recovery on-demand by specifying star
...
- **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z.
+ **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. If not in a UTC format, the timestamp is interpreted as the local time of the adapter.   
+ **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. If not in a UTC format, the timestamp is interpreted as the local time of the adapter.
```
Comment (rmuolic) [inline content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md:31]:
```suggestion
 **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime as UTC in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. <br><br>**Note:** If a timestamp is not specified in UTC, the adapter assumes it is in local time and will interpret the timestamp to UTC. For example, a timestamp of "2022-05-25T12:00" will be interpreted as "2022-05-25T12:00-07:00" if the adapter is in PDT time (UTC -7:00).
```

File: content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md (hunk 1 above)
Comment (rmuolic) [reply content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md:30]: Added a suggested change @akiaei per our comments.

**Review by rmuolic — CHANGES_REQUESTED** [review]
Added suggested changes.

**Review by aveva-jhwang — COMMENTED** [review]
Zdenek shared this approval request with me. I think the wording is a little convoluted for what we want to convey to our customers. Could we update it to say something like this (formatting didn't carry over unfortunately)? 

---------------------
EndTime | datetime| Time when the last data items are collected.

Note: Timestamps are interpreted as Coordinated Universal Time (UTC) time standard.

To specify a timestamp in UTC standard, use the following format in a curl command: YYYY-MM-DDTHH:MM:SSZ.
To specify a timestamp in the local time of the PI Adapter, use the following format in a curl command: YYYY-MM-DDTHH:MM:SS.

File: content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md [hunk 3]
```diff
@@ -27,8 +27,8 @@ The PI adapter supports performing history recovery on-demand by specifying star
...
- **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z.   
- **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z.
+ **StartTime** | `datetime` | Time when the first data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a startTime as UTC in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. <br><br>**Note:** If a timestamp is not specified in UTC, the adapter assumes it is in local time and will interpret the timestamp to UTC. For example, a timestamp of "2022-05-23T11:00" will be interpreted as "2022-05-23T11:00-07:00" if the adapter is in PDT time (UTC -7:00).
```
Comment (rmuolic) [inline content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md:30]:
```suggestion
 **StartTime** | `datetime` | Time when the first data items are collected.<br><br>**Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard.<br><br>To specify a timestamp in UTC standard, use the following format in a curl command: YYYY-MM-DDTHH:MM:SSZ.<br>To specify a timestamp in the local time of the PI Adapter, use the following format in a curl command: YYYY-MM-DDTHH:MM:SS.
```

File: content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md [hunk 4]
```diff
@@ -27,8 +27,8 @@ The PI adapter supports performing history recovery on-demand by specifying star
...
+ **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard. OSIsoft recommends the following format for entering a endTime as UTC in a curl command: *YYYY-MM-DD*T*HH:MM:SS*Z. <br><br>**Note:** If a timestamp is not specified in UTC, the adapter assumes it is in local time and will interpret the timestamp to UTC. For example, a timestamp of "2022-05-25T12:00" will be interpreted as "2022-05-25T12:00-07:00" if the adapter is in PDT time (UTC -7:00).
```
Comment (rmuolic) [inline content/main/shared-content/configuration/history-recovery/on-demand-history-recovery-configuration.md:31]:
```suggestion
 **EndTime** | `datetime`| Time when the last data items are collected.<br><br> **Note:** Timestamps are interpreted into the Coordinated Universal Time (UTC) time standard.<br><br>To specify a timestamp in UTC standard, use the following format in a curl command: YYYY-MM-DDTHH:MM:SSZ.<br>To specify a timestamp in the local time of the PI Adapter, use the following format in a curl command: YYYY-MM-DDTHH:MM:SS.
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Thank you @aveva-jhwang for your suggestion. It's a much cleaner approach w/o the need of a timestamp example. @akiaei please see my new suggestions to implement.

**Review by rmuolic — APPROVED** [review]

**Review by AnnonsonTechWriter — APPROVED** [review]

**Review by zryska — APPROVED** [review]

**Review by aveva-jhwang — APPROVED** [review]

**Pull request merged**